core/modules/user/user.module

```diff
@@ -1630,6 +1636,19 @@ function user_menu_alter(&$items) {
   }
 }
 
+/**
+ * Return the user page tabs setting.
+ */
+function user_login_page_settings() {
+  static $user_login_page_settings;
```

Comment:
Namespace aside, we should us the `backdrop_static()` model here in the event this setting needs to be reset (i.e. in tests).

```suggestion
  $user_login_appearance = &backdrop_static(__FUNCTION__);
```

Reply:
Done